manual/commands.md

```diff
@@ -0,0 +1,633 @@
+# Help
+Once installed, if you run the `meza` command all by itself, it will display *help* like the following:
+```bash
+Mediawiki EZ Admin
+
+Usage: meza COMMAND [directives]
+
+To setup a multi-server environment, do:
+
+$ meza setup env # Setup the environment, following prompts
+# Edit config as required:
+$ sudo vi /opt/conf-meza/secret/<env-name>/hosts
+$ sudo vi /opt/conf-meza/secret/<env-name>/secret.yml
+$ sudo meza deploy <env-name>
+
+Commands    Directives           Description
+---------------------------------------------------------------
+install     dev-networking       Setup networking on VM
+            monolith             Install server on this machine
```

Comment:
Does monolith imply that everything (web server, database server) is running on a single server? I don't see any commands to do the opposite, but maybe that's because I'm not understanding the difference between install and deploy in this context.